src/transformation/visitors/class/index.ts

```diff
@@ -65,28 +70,19 @@ interface ClassSuperInfo {
     extendedTypeNode?: ts.ExpressionWithTypeArguments;
 }
 
-export function transformClassDeclaration(
+function transformClassLikeDeclaration(
     classDeclaration: ts.ClassLikeDeclaration,
     context: TransformationContext,
     nameOverride?: lua.Identifier
-): OneToManyVisitorResult<lua.Statement> {
+): { statements: OneToManyVisitorResult<lua.Statement>; name: lua.Identifier } {
```

Comment:
```suggestion
): { statements: lua.Statement[]; name: lua.Identifier } {
```

Or maybe just drop this annotation, since it's just an internal helper with a simple return type that would be used only for destructuring?